supabase-ws-flutter-mobile/lib/core/di/app_providers_module.dart

```diff
@@ -30,3 +33,8 @@ extension _GetItDiModuleExtensions on GetIt {
     );
   }
 }
+
+Future<SupabaseClient> _initSupabase() => Supabase.initialize(
```

Comment:
Add todo here, we have to discuss where we should initialize... It's a ws, what do you think?